typescript/vro-scripting-api/src/api/ConfigurationElement.ts

```diff
@@ -38,10 +38,8 @@ namespace vroapi {
         * @param key
         */
         getAttributeWithKey(key: string): Attribute {
-            // const attribute = this.attributes.find(a => a.name === key);
-            const attribute: any = this.attributes != null ? this.attributes.find(a => a.name === key) : [];
-
-
+            const attribute = this.attributes.find(a => a.name === key);
```

Comment:
You can do a in-line null check with `  this.attributes?.find(a => a.name === key)`

Reply:
I appreciate the fix, the code is from BT 2.33.0, so I assume that this has always been correct.